_toc.yml

```diff
@@ -0,0 +1,61 @@
+format: jb-book
+root: README
+parts:
+  - caption: Getting started
+    chapters:
+    - file: notebooks/START_HERE
+  - caption: Failure type classifier
```

Comment:
Would it be a good idea to include the ML project README here?

Reply:
I think it would be a great idea :D added!